docs/ecosystem/wash/plugins.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Wash plugins"
+date: 2024-04-26T11:02:05+06:00
+draft: false
+sidebar_position: 3
+description: "Using wash plugins"
+---
+
+:::tip
+
+Wash plugins are available starting with version 0.28
+
+:::
+
+Wash plugins are a way to extend the functionality of wash. They are Wasm components that can be
```

Comment:
Minor tweak suggestions for this paragraph: 

"Plugins are Wasm components that can extend the functionality of `wash` with new subcommands. This functionality is experimental but enabled by default, meaning we may change the plugin API based on feedback from those using it. The component model will make it very easy to adapt an older version of a plugin to a newer version of the API, with full transparency to users and few if any changes to the plugin code."